com.github.wwmm.easyeffects.yml

```diff
@@ -0,0 +1,389 @@
+# see https://github.com/wwmm/easyeffects/blob/master/PKGBUILD for dependencies
+# If you get errors like 137, you might need pass --jobs=1 to flatpak-builder to limit cpu threads.
+  # discussion at here https://github.com/flathub/com.github.wwmm.pulseeffects/pull/53
+# todo
+# add x-checker-data
+# fix icon
+
+id: com.github.wwmm.easyeffects
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+command: easyeffects
+rename-icon: easyeffects
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=xdg-run/pipewire-0:ro
+    # LADSPA Plugin PATH
+  - --env=LADSPA_PATH=/app/lib/ladspa:/app/extensions/Plugins/ladspa
+    # LV2 Plugin PATH
+  - --env=LV2_PATH=/app/lib/lv2:/app/extensions/Plugins/lv2
+cleanup:
+  - '*.a'
+  - '*.h'
+  - '*.la'
+  - /bin/analyseplugin
+  - /bin/applyplugin
+  - /bin/listplugins
+  - /include
+  - /lib/pkgconfig
+  - /lib/python*
+  - /share/info
+
+add-extensions:
+  org.freedesktop.LinuxAudio.Plugins:
+    directory: extensions/Plugins
+    version: "20.08"
+    add-ld-path: lib
+    merge-dirs: ladspa;dssi;lv2;lxvst;vst3
+    subdirectories: true
+    no-autodownload: true
+
+modules:
+  
+  - name: easyeffects
+    buildsystem: meson
+    build-options:
+      env: # remove this variable when meson from pip is removed.
+        PYTHONPATH: /app/utils/lib/python3.8/site-packages:/app/lib/python3.8/site-packages:/usr/lib/python3.8/site-packages
+    sources:
+      - type: git
+        url: 'https://github.com/wwmm/easyeffects.git'
+        tag: v6.0.3
+        commit: 35c27c3499af42f8d1c6636358a385262caeb857
+        x-checker-data:
+          type: git
+          tag-pattern: "^v([\\d.]+)"
+    post-install:
+      - install -Dm644 -t $FLATPAK_DEST/share/licenses/com.github.wwmm.easyeffects ../LICENSE.md
+      - mkdir -pm755 /app/extensions/Plugins
```

Comment:
it doesn't matter much but usuall I use `install -d` to create it

Reply:
I tried different ways of changing this, but I never got it working properly. Since it doesn't seem to matter much I think I'll just leave it as is and resolve this.